vta/hardware/chisel/Makefile

```diff
@@ -109,7 +133,7 @@ else
   lib_path = $(vta_dir)/$(BUILD_NAME)/$(VTA_LIBNAME).so
 endif
 
-default: lint lib
```

Comment:
what was the reason for removing lint here?

Reply:
As I explained in the commit message, currently lint messes up indentation and requires manual fixes. 
It would be better in my opinion to perform lint manually after large code changes only.

Reply:
Oh I just saw your newest commits. @vegaluisjose set up the Chisel linter, he can chime in on what the best course of action is moving forward. It would be nice to keep linting the code for future submissions.

Reply:
I don't mind if you leave it there.
I personally do make lib only but then I forget, type make and indentation goes berserk, just annoying :-)

Reply:
With `--test` argument in `sbt scalafmt`, the linter would not change the code base, see PR #4555 .

Reply:
Yes, this is the right way to go.
With --test the user is left to do the actual edits.

Reply:
But unfortunately it does not work, I invite you to try yourself.
sbt scalafmt --test does change the code.
In my case it changes 6 scala files, for instance:

```diff
diff --git a/vta/hardware/chisel/src/main/scala/core/LoadUop.scala b/vta/hardware/chisel/src/main/scala/core/LoadUop.scala
index 31e0b56bd..f99ac4948 100644
--- a/vta/hardware/chisel/src/main/scala/core/LoadUop.scala
+++ b/vta/hardware/chisel/src/main/scala/core/LoadUop.scala
@@ -118,12 +118,11 @@ class LoadUop(debug: Boolean = false)(implicit p: Parameters) extends Module {
               state := sReadCmd
               xlen := xrem
               xrem := 0.U
-            }
-            .otherwise {
-              state := sReadCmd
-              xlen := xmax - 1.U
-              xrem := xrem - xmax
-            }
+            }.otherwise {
+                state := sReadCmd
+                xlen := xmax - 1.U
+                xrem := xrem - xmax
+              }
           }
         }
       }
```

Reply:
It is indeed an odd indentation choice, but I would be in favor of enabling lint even if the output is odd to have uniformity/consistency across the Chisel codebase. @liangfu @vegaluisjose any input on why the indentation of the `}` looks funky?

Reply:
I really don't mind if lint stays or not.
Given your preference I am going to put it back.
But there is definitely a bug in either scalafmt or the way sbt calls it.
I tried to change the scalafmt configuration to fix the indentation but without success...

Reply:
I think it's better to replace scala linter from `scalafmt` to  `scalatyle`, the former focus on changing the code format to a predefined style (it removed --test argument lately I think), and the later focus on checking style errors with no intention in changing the code base. I can put an update to switch the linter for scala, if you would like @tmoreau89 @vegaluisjose .

Reply:
Hey @liangfu ,

that sounds like a plan. Let's do it in a separate PR. I saw the chisel template is also using that.